src/web/app/src/main/webapp/WEB-INF/web.xml

```diff
@@ -61,7 +61,15 @@
         <param-name>contextConfigLocation</param-name>
         <param-value>classpath*:/applicationContext.xml classpath*:/applicationSecurityContext.xml</param-value>
     </context-param>
-    
+
+    <!-- do not auto init log4j, geoserver will do it -->
```

Comment:
Commented out bit... seems like you wanted to disable log4j auto init, but in the end it was no needed?

Reply:
correct; was having a lot of trouble determining order of execution. Finally just put a breakpoint in ...